docker/ubuntu1204-x86/Dockerfile

```diff
@@ -41,13 +41,13 @@ RUN mkdir -p /etc/sudoers.d
 RUN echo "jenkins    ALL=NOPASSWD: ALL" > /etc/sudoers.d/jenkins
 RUN chmod 440 /etc/sudoers.d/jenkins
 
-RUN su jenkins -c 'command curl -sSL https://rvm.io/mpapis.asc | gpg --import - && \
-  command curl -sSL https://rvm.io/pkuczynski.asc | gpg --import - && \
-  curl -L -sSL https://get.rvm.io | bash -s stable'
+RUN su jenkins -c 'command curl --insecure -sSL https://rvm.io/mpapis.asc | gpg --import - && \
+  command curl --insecure -sSL https://rvm.io/pkuczynski.asc | gpg --import - && \
+  curl --insecure -L -sSL https://get.rvm.io | bash -s stable'
```

Comment:
Call using `--insecure` is unfortunate here, other avenues to address this are in progress.  For this PR an image has been built and published for now.